docs/setup.md

```diff
@@ -2,45 +2,50 @@
 
 This document outlines the setup process for the `Superposition Platform`.
 
-## Prerequisites
+## Installation
```

Comment:
Since I assume you have specified MacOS here - I assumed you want this for Linux/MacOS - should we call that out explicitly here?

Maybe @ShubhranshuSanjeev can validate steps on Linux.

Reply:
We can mention linux/MacOS. Only the steps to install libpq will change between the two.